tests/e2e.sh

```diff
@@ -23,13 +23,14 @@ case $(uname -m) in
 	*)		ARCH="$(uname -m)";;
 esac
 
-KUBERNETES_VERSION=v1.19.1
+KUBERNETES_VERSION="v1.20.2"
```

Comment:
I actually prefer bundling all the k8s updates together in one PR.

So, can we update the go deps of k8s and the compatibility matrix ?

Reply:
Sure thing

Reply:
Thanks for letting me know! I haven't started, so I'll limit this one to kind v0.10.0 only.

Reply:
Whoops. Missed that. Thanks @lilic